doc/source/intro.rst

```diff
@@ -68,10 +68,10 @@ Features
 
 * Post-simulation analysis can be done as desired, and might for instance
   include a Griff analysis written in C++, which outputs histogram data in
-  SimpleHists files, usually followed by a final statistical analysis and
-  plot-production in Python, where all the usual tools (e.g. `Matplotlib
-  <https://matplotlib.org/>`__ and `SciPy <https://scipy.org/>`__) are
-  available.
+  :ref:`SimpleHists<sbsimplehists>` files, usually followed by a final
```

Comment:
Good catch, but I think when using `:ref:` that you need s space before the `<` character.

Reply:
I believe this is what makes the CI test fail.

Reply:
It is possible - the examples I can find on [Read The Docs](https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html) do include a space -, but I've copied the reference from some other place. Probably from [here in griff.rst](https://github.com/mctools/simplebuild-dgcode/blob/4cb880175a9462fa7226e7e29917e6fe853c3c04/doc/source/griff.rst?plain=1#L262) or [here in scans.rst](https://github.com/mctools/simplebuild-dgcode/blob/4cb880175a9462fa7226e7e29917e6fe853c3c04/doc/source/scans.rst?plain=1#L38), where the same form (no space before the < character) is used. Should I change these, and all other references without a space character as well?

Reply:
yes, those should be fixed as well - I guess you already did?